plugin/trino-iceberg/pom.xml

```diff
@@ -16,7 +16,7 @@
     <properties>
         <air.main.basedir>${project.parent.basedir}</air.main.basedir>
         <!-- Nessie version (matching to Iceberg release) must be bumped along with Iceberg version bump to avoid compatibility issues -->
-        <dep.nessie.version>0.77.1</dep.nessie.version>
+        <dep.nessie.version>0.79.0</dep.nessie.version>
```

Comment:
This version comes from https://github.com/apache/iceberg/blob/apache-iceberg-1.5.0/gradle/libs.versions.toml#L70. `0.77.1` is the expected version in Iceberg 1.5.0. 

We should fix the comment at L18 if we don't follow the version.

Reply:
We were not following this comment in the past

Reply:
I've updated the comment